test/entity.test.js

```diff
@@ -47,22 +47,16 @@ describe('handles block entities', () => {
         const simplified = nbt.simplify(tag)
         sign.entity = tag
 
-        if (registry.type === 'pc') {
-          assert.deepEqual(simplified.Text1, JSON.stringify(sign.blockEntity.Text1.json))
-        }
-
-        if (registry.type === 'bedrock') {
-          assert.deepEqual(simplified.Text, sign.blockEntity.Text)
-        }
+        assert.deepEqual(simplified.Text, sign.blockEntity.Text)
       })
 
       it('.signText works on ' + version, () => {
         const sign = Block.fromStateId(registry.blocksByName.standing_sign?.defaultState || registry.blocksByName.oak_sign?.defaultState)
         sign.signText = ['Hello', 'World']
 
         if (registry.type === 'pc') {
-          assert(sign.blockEntity.Text1.toString() === 'Hello')
-          assert(sign.blockEntity.Text2.toString() === 'World')
+          if (registry.version['>=']('1.20')) assert(sign.getSignText()[0] === 'Hello\nWorld')
+          else assert(sign.getSignText()[0] === 'Hello\nWorld\n\n')
```

Comment:
The reason it's like this, is Minecraft changed to sending an array of lines instead of line1, line2 etc., so to me it makes more sense to display the number of lines it received for new signs. So if 2 lines are set, it displays 2 lines.  If the server sends 4 lines, then getSignText will return 4 lines. What do you think?

Reply:
getSignText()/.signText is supposed to return parsed plaintext, so it removes all the chat formatting (like colors and underlining) from the message. So it's expected the output will be the same regardless of the version. Doing a .trim() operation internally seems like it would fix this padding issue.

Reply:
Personally I think it should return a string with the number of lines that have been set by the server, which will probably be 4 in most cases and always 4 for versions before 1.20. But if that's how you think it should act, sure